docs/source/specs/openapi.yml

```diff
@@ -660,18 +660,24 @@ definitions:
         enum:
           - asc
           - desc
+      delta:
+        type: string
+        enum:
+          - asc
+          - desc
+      account_alias:
+        type: string
+        enum:
+          - asc
+          - desc
     example: { 'total': 'asc' }
     description: The ordering to apply to the report. Default is ascending order for the data.
   ReportDelta:
     type: object
     properties:
       cost:
```

Comment:
shouldn't this be `delta`?

Reply:
I'm not sure I understand what you're asking.

Reply:
https://github.com/project-koku/koku/blob/master/docs/source/specs/openapi.yml#L668

Report Delta, shouldn't the value be `delta` not `cost`

Reply:
Good catch. You're right.